docs/gettingstarted/index.md

```diff
@@ -67,7 +67,7 @@ base.archivesName = 'mymod'
 
     ```gradle
     // In some build.gradle
-    base.archivesName = 'mymod'
+    archivesBaseName = 'mymod'
```

Comment:
Why did you change this?

Reply:
Because it was a mistake. It's within a note saying that the Forge MDK was using `archivesBaseName`. However, since we're no longer doing that. I can just remove the entire note itself.